docs/docs/guides/cookbooks/index.md

```diff
@@ -0,0 +1,7 @@
+# Weave Cookbooks
+
+Weave Cookbooks provides a collection of byte-sized recepies of LLM applications showcasing effective use of different LLM tools and their integrations with Weave for tracing, versioning, and evaluations to solve a real-world usecase. In these cookbooks you will discover practical insights, step-by-step tutorials, and best practices to effectively utilize Weave in your LLM applications.
```

Comment:
Hi @scottire, does this serve as a good definition of a cookbook?

Reply:
I think the question is: 
How are cookbooks different to tutorials? 
One suggestion would be that we group them by complexity where tutorials assume zero knowledge of the APIs and introduce everything to the user, where cookbooks can assume basic knowledge of the API, and provide more robust examples of use cases. cc: @morganmcg1